api/app/Http/Controllers/ThreadController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Thread;
+use Illuminate\Http\Request;
+
+class ThreadController extends Controller
+{
+    //
+    public function fetchThread(Request $request, $id) 
+    {
+        $thread = Thread::where('announcement_id', $id)->get();
+
+        return response()->json($thread);
```

Comment:
Can we also fetch the participants of the thread? Same what we did on the announcement page sir 🙇